docs/Running-Tachyon-on-EC2-Yarn.md

```diff
@@ -138,7 +138,7 @@ ApplicationMaster for Tachyon.
 ```bash
 $ cd /tachyon
 $ ./bin/tachyon-stop.sh
-$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn -DskipTests
+$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn -DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.skip
```

Comment:
if it only speeds up the build, I feel it might be better to present both commands
`$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn` (as default) and also mention to speed up, we could use
`$ mvn clean install -Dhadoop.version=2.4.1 -Pyarn -DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.skip`.

My concern is if we present the latter one as default, people may tend to think all flags are required to make the source build.